darshan-util/pydarshan/darshan/tests/test_plot_dxt_heatmap.py

```diff
@@ -98,7 +98,7 @@ def test_set_x_axis_ticks_and_labels(
         data = [[4, 1.03378843, 1.03387713, 0], [4000, 1.04216653, 1.04231459, 0]]
         cols = ["length", "start_time", "end_time", "rank"]
         agg_df = pd.DataFrame(data=data, columns=cols)
-        runtime = 1
+        runtime = 2
```

Comment:
what's going on here? changing the behavior for the synthetic data set because you always round up under the hood now or?